README.md

```diff
@@ -28,31 +28,55 @@ We want to hear from you! Are you interested in building or currently building i
 
 ![Chat screen](docs/chatscreen.png)
 
+## Application architecture
+
+- **User interface** - The application’s chat interface is a [Blazor WebAssembly](https://learn.microsoft.com/aspnet/core/blazor/) static web application. This interface is what accepts user queries, routes request to the application backend, and displays generated responses.
```

Comment:
Is "static web application" the best descriptor here? Should we state it's a "Blazor Web Assembly application" instead?

Reply:
Good call. It's technically hosted inside the Web service today.